### DISCUSSION TITLE
Loan amount

### DISCUSSION THREAD

**Discussion by nitins11** [ONDC-Official/ONDC-FIS-Specifications#15] [General]
I remember there was a question earlier if the "loan amount" value would be taken by the buyer app and the answer was No. But the Swagger Flow : One of the select calls has a loan amount form. Where is that being used. Please help clarify our understanding.

**Comment by ankitagarwaladobe** [comment]
IIUC
As of now the loan amount from the borrower is not passed in the **search** request but once the borrower shows interest in a particular offer then the provider platform responds with detailed information and the necessary application form for the selected personal loan service.
At this point of time the borrower can ask for specific amount and tenure and then the lender will respond with updated detailed information.

**Reply by nitins11** [reply]
Noted. Thanks!

**Reply by Antriksh-Parmar** [reply]
@92shreyansh you might want to clarify. 
Also, borrower can only reduce the loan amount and can not change the loan tenure.

**Reply by ankitagarwaladobe** [reply]
@Antriksh-Parmar 
This is a common practice to change both the loan amount and loan tenure by the borrower. 
I agree, that tenure may go out of range and may not be acceptable to the borrower but the borrower should be able to change both.

I will start another thread on this:
Ideally, the lender should return multiple combinations in 'on_select' response.
For example:
Loan range - 1L to 5L 
Tenure - 12-24 months
Interest - 8%


Loan range - 1L to 5L 
Tenure - 6-11 months
Interest - 9%

and so on.

this will allow BAPs to handle a few cases at their end and enhance user experience.

**Reply by Antriksh-Parmar** [reply]
@ankitagarwaladobe Yes, I understand. The idea is to allow the lender to come back with the best possible offer for the borrower from the amount and tenure perspective. But let me have another internal discussion on this and see the best way to have a flexible tenure for approved loan amount.

**Reply by nitins11** [reply]
Lenders may not appreciate too much variation in their credit policies. Few have flexible systems today to create dynamic credit products as it impacts their downstream systems such as LMS, Reporting etc. Their credit policies/systems are usually created on the loan amount + tenor + loan type. A change in any of these params is actually a new product for them.

**Reply by ankitagarwaladobe** [reply]
@nitins11 
I am proposing this as part of the on_select response which is not the final offer. The lender can definitely return multiple offers and then based on the user selection can return the final policy in on_init/on_confirm response.
It will definitely be better than making multiple api calls for change in amount and tenure. This may add a some latency to the user experience.
